cmd/crane/rebase.md

```diff
@@ -1,68 +1,85 @@
-This command rewrites an image's manifest to replace layers in a base image
-with layers in another version of that base image. It does so entirely with API
-calls to the registry, so it doesn't have to download or upload full layer
-blobs, if it can be avoided.
+### This code is experimental and might break you if not used correctly.
+
+The `rebase` command efficiently rewrites an image to replace the base image it
+is FROM with a new base image.
+
+# TODO: diagram
```

Comment:
This appears as a gigantic header, not a comment :)

Can we do this for this change?